hazelcast/src/test/java/com/hazelcast/jet/impl/JetClientInstanceImplTest.java

```diff
@@ -51,11 +56,77 @@ public void given_singleMapOnMember_when_getDistributedObjectsCalled_then_Return
         // Then
         assertFalse(objects.isEmpty());
         DistributedObjectInfo info = objects.stream()
-                                            .filter(i -> mapName.equals(i.getName()))
-                                            .findFirst()
-                                            .orElseThrow(AssertionError::new);
+                .filter(i -> mapName.equals(i.getName()))
+                .findFirst()
+                .orElseThrow(AssertionError::new);
```

Comment:
Formatting again.

Reply:
I am using IntelliJ's default formatting. Event if I pull the code left, IntelliJ will make it this was next time